### PR TITLE
Added Nigxht

### PR DESCRIPTION
**Pull request by Danidev819**
### General:

* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:

* [x] My formatting matches the pre-existing formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping with my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:

* [x] My image is a `.png` file.
* [x] I uploaded the image in the `logos` directory.
* [x] My image is transparent & square (1:1 aspect ratio).
* [x] My image is a minimum of `512` pixels in width and height.
* [x] My images image name matches its mapping id.

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1648,5 +1648,14 @@
 	"addresses": [
 	    "zyro.gg"
 	]
+    },
+    {
+        "id": "nigxht",
+        "name": "Nigxht Network",
+        "addresses": [
+            "nigxht.net",
+	    "mc.nigxht.us",
```
Comment (GrowlyX) [inline servers.json:1657]: Please look over the checklist.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json (hunk 1 above)
Comment (Danidev819) [reply servers.json:1657]: What happens

File: servers.json [hunk 2]
```diff
@@ -1653,9 +1653,14 @@
         "id": "nigxht",
         "name": "Nigxht Network",
         "addresses": [
-            "nigxht.net",
-	    "mc.nigxht.us",
-	    "play.nigxht.us"
+            "nigxht.net"
```
Comment (GrowlyX) [inline servers.json:1656]: You're allowed to have your other .us domain, we just don't want sub-domains.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json (hunk 2 above)
Comment (Danidev819) [reply servers.json:1656]: Now only have nigxht.net

File: servers.json [hunk 3]
```diff
@@ -1650,10 +1650,17 @@
 	]
     },
     {
+        "id": "nigxht",
+        "name": "Nigxht Network",
+        "addresses": [
+            "nigxht.net"
+        ]
+
```
Comment (GrowlyX) [inline servers.json:1658]: Please add a closing bracket here.

File: servers.json [hunk 4]
```diff
@@ -1650,10 +1650,17 @@
...
 	"id": "techcraft",
 	"name": "TechCraft Network",
 	"addresses": [
 	    "techcraftmc.com"
 	]
+
```
Comment (GrowlyX) [inline servers.json:1664]: Please remove this unnecessary indent.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Review by GrowlyX — APPROVED** [review]

File: servers.json [hunk 5]
```diff
@@ -1650,6 +1650,12 @@
 	]
     },
     {
+        "id": "nigxht",
+        "name": "Nigxht Network",
+        "addresses": [
+            "nigxht.net"
+        ]
+    },
 	"id": "techcraft",
```
Comment (GrowlyX) [inline servers.json:1659]: You have to add back the starting bracket for techcraft.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Review by GrowlyX — APPROVED** [review]

**Pull request merged**